packages/test/mocha-test-setup/api-report/mocha-test-setup.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-/// <reference types="mocha" />
```

Comment:
IIRC this is some incremental API-extractor bug?

Reply:
Ya, I'll rever